.github/workflows/ci.yaml

```diff
@@ -279,3 +285,11 @@ jobs:
           just sql-logic-tests --rpc-test --exclude '*/tunnels/ssh' 'sqllogictests_mongodb/*'
           just sql-logic-tests --rpc-test --exclude '*/tunnels/ssh' 'sqllogictests_mysql/*'
           just sql-logic-tests --rpc-test --exclude '*/tunnels/ssh' 'sqllogictests_postgres/*'
+          
+          echo "-------------------------- REMOTE DATA STORAGE TESTS --------------------------------"
+          # Test using a remote object store for storing databases and catalog; For now only on MinIO (S3)
+          just sql-logic-tests --location http://localhost:9000 \
+              --option access_key_id=$MINIO_ACCESS_KEY \
+              --option secret_access_key=$MINIO_SECRET_KEY \
+              --option bucket=$MINIO_BUCKET \
+              'sqllogictests_native/*' 
```

Comment:
We probably want to include some more tests that are heavy on DDL & DML statement, so as to exercise the remote object store as much as possible. What would be some good paths to add here?

Reply:
I think we should run the `sqllogictests/*` tests as well using this. Most of the DDL/DML tests lie there!

Reply:
Great, thanks for the info!